src/App.tsx

```diff
@@ -1,25 +1,18 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import {RenderRoutes} from "./navigation/Router"; 
```

Comment:
it's good to be consistent when using spaces in the objects.
so that, it will be the same within a project.

```suggestion
import { RenderRoutes } from "./navigation/Router"; 
```

Reply:
also, it's good to have this thing configured on a lint level